provisioning/buffer/kubernetes/templates/etcd.yaml

```diff
@@ -8,7 +8,7 @@ resources:
   limits:
     memory: "2000Mi"
 
-# Run defragmentation each 2 hours
+# Run defragmentation each 30 minutes
 defragmentation:
   cronjob:
-    schedule: "0 */2 * * *"
+    schedule: "*/30 * * * *"
```

Comment:
Ked z etcd nieco zmazes, tak sa realne miesto na disku uvolni, az ked sa spusti `defragmentation`.
- nevyhoda je, ze sa vsetko na `~8s` "zastavi"
- dalo by sa to defragmentovat aj zlozitejsie, ze sa jeden node vypne, defragmentuje vypnuty, zapne
  - ale to sme zatial neskusali, malo by sa to dat nascriptovat

Znizil som interval defragmentacie, aby nam nedoslo miesto na disku vramci testovania.